src/index.js

```diff
@@ -0,0 +1,97 @@
+const EventEmitter = require('events')
+const DIDIPFSHelper = require('./modules/didIpfsHelper')
+const APIClient = require('./modules/http')
+const Persistence = require('./modules/persistence')
+const Vendor = require('chlu-ipfs-support/src/modules/vendor')
+const ReviewRecords = require('chlu-ipfs-support/src/modules/reviewrecords')
+const Protobuf = require('chlu-ipfs-support/src/modules/protobuf')
+const Logger = require('chlu-ipfs-support/src/utils/logger')
+const Storage = require('chlu-ipfs-support/src/utils/storage')
+const { get } = require('lodash')
+const path = require('path')
+
+
+class ChluAPIClient {
+
+    constructor(options = {}) {
+        const defaultDirectory = path.join(get(process, 'env.HOME', ''), 'chlu-api-client')
+        this.directory = options.directory || defaultDirectory
+        this.enablePersistence = get(options, 'enablePersistence', true)
+        // Some of the Chlu full node modules are necessary here too
+        this.storage = Storage
+        this.logger = options.logger || Logger
+        this.events = new EventEmitter()
+        this.persistence = new Persistence(this)
+        this.protobuf = new Protobuf(this)
+        this.reviewRecords = new ReviewRecords(this)
+        this.vendor = new Vendor(this)
+        // Some are specific versions for the api client
+        this.didIpfsHelper = new DIDIPFSHelper(this)
+        this.api = new APIClient(
+            options.queryApiUrl || '/',
+            options.publishApiUrl || '/'
+        )
+    }
+
+    async start() {
+        await this.persistence.loadPersistedData()
+        await this.didIpfsHelper.start()
+    }
+
+    async stop() {
+        await this.persistence.persistData()
+    }
+
+    async readReviewRecord(multihash, options) {
+        return await this.api.readReviewRecord(multihash, options)
+    }
+
+    async storeReviewRecord(reviewRecord, options = {}) {
+        // prepare the review record
+        // this will sign it as both customer and issuer. The server can then replace the issuer signature with its own
+        // the 'validate' option is forced to false because the API Client cannot validate.
+        const bitcoinTransactionHash = get(options, 'bitcoinTransactionHash', null)
+
+        reviewRecord.issuer = await this.api.getPublisherDID() // Authorize server to issue review
```

Comment:
Yeah, we should take away any reference to PoPR backed reviews in this store method.